### ISSUE TITLE
Where can I find the original single hop questions?

### ISSUE THREAD

**Issue by sagnik** [StonyBrookNLP/musique#5]
Hi

Thanks for the great work!

I am looking for the original single-hop questions that were combined. For example, in the first answerable validation question, we have this:

```
 'question_decomposition': {'id': [460946, 294723],
  'question': ['Green >> performer', '#1 >> spouse'],
  'answer': ['Steve Hillage', 'Miquette Giraudy'],
  'paragraph_support_idx': [10, 5]},
```

Can we use these ids to trace back the original questions from the datasets?

**Comment by HarshTrivedi** [comment]
These IDs refer to the unique identifiers of our compiled dataset (box 2 in Figure 2), and not original datasets. However, if you really want, it is possible to find the original QA instance from source datasets using the similarity-based match based on the question, paragraph text, and answer string from the decomposition step. I am curious though, why do you need it as you already have information about Q, C, and A from the decomposition step?

On a related note, if you are planning to train models on single-hop questions from constituting datasets, please make sure to see the Usage Caution note on the readme.

**Comment by sagnik** [comment]
```
However, if you really want, it is possible to find the original QA instance from source datasets using the similarity-based match based on the question, paragraph text, and answer string from the decomposition step. 
```

Thanks, I imagined doing something like this, but not before confirming. You do have the single hop questions for the dev and test set (which I imagine will reduce the search space a bit), any chance you have them for the training set?

```
I am curious though, why do you need it as you already have information about Q, C, and A from the decomposition step?
```
I want to only look at the multihop questions generated from SQuAD. If there is an indicator of the original dataset somewhere, that would solve my problem for now.

Also, off the topic, is this [HF dataset](https://huggingface.co/datasets/bdsaglam/musique) an official release? If yes, this probably can be included in the README.

**Comment by sagnik** [comment]
sent.

**Comment by HarshTrivedi** [comment]
Shared the files. Closing this now. Feel free to open if you've more questions.
